Services/CO.CDP.EntityVerification/UseCase/LookupIdentifierUseCase.cs

```diff
@@ -0,0 +1,50 @@
+using CO.CDP.EntityVerification.Model;
+using CO.CDP.EntityVerification.Persistence;
+using System.Collections.Generic;
+using static CO.CDP.EntityVerification.UseCase.LookupIdentifierUseCase.LookupIdentifierException;
+
+namespace CO.CDP.EntityVerification.UseCase;
+
+public class LookupIdentifierUseCase(IPponRepository repo) : IUseCase<LookupIdentifierQuery, IEnumerable<Model.Identifier>?>
```

Comment:
Unused property repo.

Reply:
I wouldn't add anything that's not needed for the task at hand. You never know, when it comes to the implementation, you'll know more and might decide to go different direction.

Also, any static analysis warnings on the PR, are later also reported on other PRs. This is going to be annoying, up until we learn to ignore any warnings.  Broken windows theory and all that.

<img width="1537" alt="image" src="https://github.com/user-attachments/assets/e00eea14-7a02-471b-a397-50d1d6c38581">